code/game/gamemodes/miniantags/bot_swarm/swarmer.dm

```diff
@@ -158,9 +158,9 @@
 ////CTRL CLICK FOR SWARMERS AND SWARMER_ACT()'S////
 /mob/living/simple_animal/hostile/swarmer/AttackingTarget()
 	if(!isliving(target))
-		target.swarmer_act(src)
+		return target.swarmer_act(src)
```

Comment:
Does this mean all AttackingTarget() calls everywhere are expected to return something now?

Reply:
Yes.